### PR TITLE
MagicaVoxel: Update to version 0.99.6.4, remove 32-bit

### PR DESCRIPTION
**Pull request by phpcitizen**
64-bit link correction, cannot find the 32-bit on their github releases.

### CODE REVIEW

File: bucket/MagicaVoxel.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
     "license": "Unknown",
```
Comment (issaclin32) [inline bucket/MagicaVoxel.json:5]:
```suggestion
    "license": {
        "identifier": "Freeware",
        "url": "https://ephtracy.github.io/"
    },
```

**Comment by issaclin32** [comment]
Also modify *checkver/autoupdate* as the following. Otherwise there will be errors when *Scoop* automatically updates the package in the future.

```
    "checkver": {
        "url": "https://github.com/ephtracy/ephtracy.github.io/releases",
        "regex": "MagicaVoxel-([\\d.]+)-win64\\.zip"
    },
    "autoupdate": {
        "architecture": {
            "64bit": {
                "url": "https://github.com/ephtracy/ephtracy.github.io/releases/download/$matchHead/MagicaVoxel-$version-win64.zip",
                "extract_dir": "MagicaVoxel-$version-win64"
            }
        }
    }
```
For built-in vairables such as `$matchHead`, check: https://github.com/ScoopInstaller/Scoop/wiki/App-Manifest-Autoupdate

**Comment by issaclin32** [comment]
`https://github.com/ephtracy/ephtracy.github.io/releases/download/0.99.6/MagicaVoxel-0.99.6.4-win64.zip` is not accesible with *Scoop*, but works fine on *checkver.ps1*. Probably related to client identifier or referer. Will investigate later.

I would be grateful if you can provide some information:
@rasa @r15ch13 @niheaven @pratikpc @rashil2000

**Comment by phpcitizen** [comment]
> `https://github.com/ephtracy/ephtracy.github.io/releases/download/0.99.6/MagicaVoxel-0.99.6.4-win64.zip` is not accesible with `scoop install`, but works fine on `checkver.ps1`.
> 
> Probably related to client identifier or referer. Will investigate later.


@issaclin32 Can you please resolve this from your side, I am having a problem. Feel free to Pull request again. Thanks.

**Comment by issaclin32** [comment]
implemented in https://github.com/ScoopInstaller/Extras/commit/edb84f5ecbfbff8d80a1e77569a9188689c4daba

**Pull request closed without merging**